Automattic/wp-calypso
# Jetpack Professional plan: promote Jetpack scan from inferior hidden feature to hidden feature

#### Changes proposed in this Pull Request

* Jetpack Professional monthly is configured with Jetpack Scan as an inferior hidden feature, while Jetpack Professional yearly has it as a hidden feature. This PR changes the monthly version of the plan to promote Jetpack Scan from inferior hidden feature to hidden feature.

#### Testing instructions

* Run this PR.
* Selected a site that has Jetpack Professional **monthly**.
* Visit `/plans/:site`.
* Verify that Jetpack Scan is featured as included in the plan.

Fixes 1169247016322522-as-1193677921810244

#### Demo
##### Before
-----
![image](https://user-images.githubusercontent.com/3418513/93141332-5fcf6a00-f6ba-11ea-88d6-2760ef0657ff.png)

##### After
-----
![image](https://user-images.githubusercontent.com/3418513/93141288-4e865d80-f6ba-11ea-8a06-73c5fcc10786.png)
